lib/apiservers/portlayer/swagger.yml

```diff
@@ -859,3 +920,46 @@ definitions:
       state:
         type: string
         enum: ["RUNNING", "STOPPED"]
+  VolumeRequest:
+    type: object
+    required:
+      - Name
+      - Store
+      - Driver
+    properties:
+      Name:
+        type: string
+      Driver:
+        type: string
+      DriverArgs:
+        type: object
+        additionalProperties:
+          type: string
+      Store:
+        type: string
+      Metadata:
+        type: object
+        additionalProperties:
+          type: string
+  VolumeResponse:
+    type: object
+    required:
+      - Name
+      - SelfLink
+      - Store
+      - Driver
```

Comment:
this should essentially just return "vsphere" as the value for driver. Might be worthwhile just to have interpretation code for that in the personality and remove driver from here as well. The portlayer does not need to know that a driver specification is necessary.